2/README.md

```diff
@@ -3,80 +3,178 @@ domain: rfc.tango-controls.org
 
-Its model can be represented as a defined tree which each elements are from a defined types: Class, Device, Property, Attribute, Command following the rules below:
+* The Device is a distributed object which SHALL be accessible locally or via network.
+* The Device SHALL be an instance of one Device Class, see [RFC-9]()
+* The Device MAY have one or several Property, called Device Property, see [RFC-5]()
+* The Device MAY have one or several Attribute, see [RFC-4]() 
+* The Device MAY have one or several Pipe
+* The Device MAY have one or several Command, see [RFC-3]()
 
+* The Device SHALL have one Init command
 
-* The Device is a distributed object which SHALL be accessed locally or via network.
-* The Device SHALL be an instance of one Class
-* The Device MAY have one or several Property, called Device Property
-* The Device MAY have one or several Attribute
 * The Device SHALL have one State attribute 
 * The Device SHALL have one Status attribute
-* The Device MAY have one or several Command
-* The Device SHALL have one unique identifier which represents its Name
+* The Device SHALL have one State command
+* The Device SHALL have one Status command
+
 
-* The Class MAY have one or more Device instance
-* The Class MAY have one or several Property, called Class Property
+**Note**: Future specification may remove 'SHALL have' requirements for State and Status commands.
+
+* The Device SHALL have one unique identifier which represents its Name
 
 ### Naming convention
+
 * The Device's Name SHALL use the following convention:
+
 ``` ABNF
 device-name = domain "/" family "/" member
 domain = 1*VCHAR
 family = 1*VCHAR
 member = 1*VCHAR
 ```
+
+### Device Class
+
+A Device Class is an association of a list of Device Class Properties, a list of Attributes, a list of Pipes, 
+a list of Commands, a list of Device Properties with a Device Class Name.  
+
+* The Device Class Name SHOULD reflect its instances application context. 
+* The Device Class Name SHALL use the following convention:
+
+``` ABNF
+device-class-name = 1*VCHAR
+```
+
+* The Device Class MAY have one or more Device instance
+* The Device Class MAY have one or several Property, called Class Property
+
+* The Device SHALL expose (provide access to) all Attributes, Pipes and Commands defined by its Device Class.
+ 
+### Device Interface
+
+The Device Interface is a list of Attributes, Pipes and Commands provided by a Device.
+
+* The Device MAY expose Attributes and/or Commands not defined by its Device Class. These are called 
+  Dynamic Attributes and/or Dynamic Commands respectively, see [RFC-XX]().
+
+* The Dynamic Attributes and Dynamic Commands MAY be added to the interface during Device Initialisation phase and/or 
+  Device Operation phase. 
+
+
+### Device lifecycle
+
+The Device lifecycle is:
+* Device Initialisation phase, when Device object is created and made visible in a Tango Controls System (see [RFC-1](), [RFC-8]())
```

Comment:
Device Initialisation phase, when Device object is created and made visible (exported) in a Tango...

Reply:
Thanks, @lorenzopivetta, I will rephrase the paragraph to use wording Exported and Not Exported.